src/modules/fundingManager/bondingCurveFundingManager/RedeemingBondingCurveFundingManagerBase.sol

```diff
@@ -108,15 +118,18 @@ abstract contract RedeemingBondingCurveFundingManagerBase is
     /// to be possible. No further functionality is implemented which would manages the outflow of
     /// collateral, e.g., restricting max redeemable amount per user, or a redeemable amount which
     /// differes from the actual balance.
+    /// Throws an exception if `_depositAmount` is zero or if there's insufficient collateral in the
+    /// contract for redemption.
     /// @param _receiver The address receiving the redeem amount.
     /// @param _depositAmount The amount of tokens being sold by the receiver.
+    /// @param _minAmountOut The minimum acceptable amount the user expects to receive from the transaction.
     /// @return redeemAmount The amount of tokens that are transfered to the receiver in exchange for _depositAmount.
-    /// Throws an exception if `_depositAmount` is zero or if there's insufficient collateral in the
-    /// contract for redemption.
-    function _sellOrder(address _receiver, uint _depositAmount)
-        internal
-        returns (uint redeemAmount)
-    {
+    /// @return feeAmount The amount of collateral token subtracted as fee
+    function _sellOrder(
+        address _receiver,
+        uint _depositAmount,
+        uint _minAmountOut
+    ) internal returns (uint redeemAmount, uint feeAmount) {
```

Comment:
Again for my understanding:
Do we take a fee from users for selling and buying in this module?

Reply:
Yes, as it is right now we do. This could be something to think about in relation to our fee structure. In an hypothetical V2, we could have the "sell" function create payment orders, and thus go through our regular fee system.